bigbluebutton-html5/imports/ui/components/app/container.jsx

```diff
@@ -34,6 +34,7 @@ const defaultProps = {
   navbar: <NavBarContainer />,
   actionsbar: <ActionsBarContainer />,
   media: <MediaContainer />,
+  children: null,
```

Comment:
I think this line can be removed - we don't use children in this file.

Reply:
:+1: